x/feedistribution/types/msg_update_params.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ sdk.Msg = &MsgUpdateParams{}
+
+// ValidateBasic does a sanity check on the provided data.
+func (m *MsgUpdateParams) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(m.Authority); err != nil {
+		return errorsmod.Wrap(err, "invalid authority address")
+	}
+
+	if err := m.Params.Validate(); err != nil {
```

Comment:
This is only stateless validation. When processing the message, stateful validation must be performed: check whether the provided epoch is registered in x/epochs.